installation/linux.md

```diff
@@ -57,36 +61,47 @@ Decide between two options:
   echo 'deb http://dl.bintray.com/openhab/apt-repo2 testing main' | sudo tee /etc/apt/sources.list.d/openhab2.list
   ```
 
+  Resynchronize the package index:
+
+  ```shell
+  sudo apt-get update
+  ```
+
+  Finally install openHAB2 as either an offline or online distribution. The **offline distribution** comes with all add-ons, the **online distribution** will install additional add-ons from the internet on request.
+  
+  ```shell
+  sudo apt-get install openhab2-offline
+  # OR #
+  sudo apt-get install openhab2-online
+  ```
+
 * **Snapshot Release**
 
   Add the **openHAB 2 Unstable Repository** to your systems apt sources list (will overwrite the existing `openhab2.list`):
 
   ```shell
   echo 'deb http://dl.bintray.com/openhab/apt-repo2 unstable main' | sudo tee /etc/apt/sources.list.d/openhab2.list
   ```
-  
-Additionally, you need to add the openHAB 2 Bintray repository key to your package manager:
 
-```shell
-wget -qO - 'https://bintray.com/user/downloadSubjectPublicKey?username=openhab' | sudo apt-key add -
-```
+  Resynchronize the package index:
 
-Scan the newly added repository and resynchronize the package index:
+  ```shell
+  sudo apt-get update
+  ```
 
-```shell
-sudo apt-get update
-```
+  Now **install** openHAB with the following command:
 
-Finally install openHAB 2 as either offline or online distribution.
-The **offline distribution** is full blown and comes with all add-ons, the **online distribution** will install additional add-ons on request from the internet.
+  ```shell
+  sudo apt-get install openhab2
+  ```
 
-```shell
-sudo apt-get install openhab2-offline
-# or
-sudo apt-get install openhab2-online
-```
+  The lastest snapshots will download add-ons on request, if you plan on disconnecting your machine from the internet then you will want to also install the addons package:
 
-If everything went well, start openHAB and register it to be automatically executed at system startup:
+  ```shell
+  sudo apt-get install openhab2-addons
```

Comment:
Why do you think it's related to the migration tutorial?
I wouldn't hide the existence of the package in the official documentation. If you can't trust the official documentation to tell you everything, that's a bad sign.

Wdyt?